api/src/main/scala/uk/ac/wellcome/platform/api/finatra/exceptions/CaseClassMappingExceptionMapper.scala

```diff
@@ -24,13 +24,15 @@ class CaseClassMappingExceptionWrapper @Inject()(
 
   val contextUri: String = s"${apiScheme}://${apiHost}${apiContext}"
 
-  override def toResponse(request: Request, e: CaseClassMappingException): Response = {
+  override def toResponse(request: Request,
```

Comment:
`scalafmt`!

Reply:
Fixing it!

Reply:
(And working out why it didn't run on my PR)